tests/unit/modules/test_virtualenv_mod.py

```diff
@@ -17,6 +17,8 @@
 from tests.support.mock import MagicMock, patch
 from tests.support.unit import TestCase
 
+VENV_BIN = f"{sys.executable} -m venv"
```

Comment:
The module is modified to use virtualenv, yet we're using venv for the tests. Is that right?